### ISSUE TITLE
apt: Unknown error executing apt-key

### ISSUE THREAD

**Issue by Garfield550** [AOSC-Dev/aosc-os-core#20]
**DUPLICATE** **FIXED** AOSC-Dev/aosc-os-abbs#128 **WRONG PLACE** 

```
aosc [ ~ ] ! sudo apt update && sudo apt dist-upgrade
Ign:1 http://mirrors.ustc.edu.cn/anthon/os3-next/os3-dpkg  InRelease
Get:2 http://mirrors.ustc.edu.cn/anthon/os3-next/os3-dpkg  Release [830 B]
Get:3 http://mirrors.ustc.edu.cn/anthon/os3-next/os3-dpkg  Release.gpg [473 B]
Ign:3 http://mirrors.ustc.edu.cn/anthon/os3-next/os3-dpkg  Release.gpg
Ign:4 https://repo.aosc.io/os3-next/os3-dpkg  InRelease
Get:5 https://repo.aosc.io/os3-next/os3-dpkg  Release [830 B]
Get:6 https://repo.aosc.io/os3-next/os3-dpkg  Release.gpg [473 B]
Ign:6 https://repo.aosc.io/os3-next/os3-dpkg  Release.gpg
Ign:7 http://mirrors.anthonos.org/anthon/os3-next/os3-dpkg  InRelease          
Get:8 http://mirrors.anthonos.org/anthon/os3-next/os3-dpkg  Release [830 B]
Get:9 http://mirrors.anthonos.org/anthon/os3-next/os3-dpkg  Release.gpg [473 B]
Err:9 http://mirrors.anthonos.org/anthon/os3-next/os3-dpkg  Release.gpg
  Unknown error executing apt-key
Reading package lists... Done 
W: GPG error: http://mirrors.ustc.edu.cn/anthon/os3-next/os3-dpkg  Release: Unknown error executing apt-key
E: The repository 'http://mirrors.ustc.edu.cn/anthon/os3-next/os3-dpkg  Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: GPG error: https://repo.aosc.io/os3-next/os3-dpkg  Release: Unknown error executing apt-key
E: The repository 'https://repo.aosc.io/os3-next/os3-dpkg  Release' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.
W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://mirrors.anthonos.org/anthon/os3-next/os3-dpkg  Release: Unknown error executing apt-key
```

**Comment by bobcao3** [comment]
I find another user also have this problem. So it's a little common

**Comment by Artoria2e5** [comment]
Dup AOSC-Dev/aosc-os-abbs#128

Upstream bug, will be fixed in apt 1.1.4. For now, grab a copy of the debianutils deb and install it.

**Comment by Artoria2e5** [comment]
By the way, apt is not an AOSC core package. Check https://github.com/AOSC-Dev/aosc-os-core/find/master before you report.
